### PR TITLE
chore(git-templates): feature request and accessbility issue templates

### PR DESCRIPTION
**Pull request by annawen1**
Closes #[4776](https://github.com/carbon-design-system/ibm-products/issues/4776)
Closes #[4777](https://github.com/carbon-design-system/ibm-products/issues/4777)

New git templates for accessibility issues and for feature requests

#### How did you test and verify your work?

You can view the form in preview mode on my fork
accessibility: https://github.com/annawen1/ibm-products/tree/feat/feature-and-accessbility-templates/.github/ISSUE_TEMPLATE/accessilibity-issue.yaml
feature request: https://github.com/annawen1/ibm-products/tree/feat/feature-and-accessbility-templates/.github/ISSUE_TEMPLATE/feature-request.yaml

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        the solution if possible. Design specs are highly encouraged.'
+    validations:
+      required: true
+  - type: textarea
+    id: design
+    attributes:
+      label: Design link
+      description: Provide link to design of example of existing component
+    validations:
+      required: true
```
Comment (elycheea) [inline .github/ISSUE_TEMPLATE/feature-request.yaml:42]: Hmm, thoughts on whether we can make this one optional? We sometimes get enhancement requests specifically from development (truncation, string to node), etc. But maybe having a design that supports it still makes sense. 🤔

**Review by elycheea — COMMENTED** [review]
One question on design link requirement for enhancement requests. 👀

Also I’m not _totally_ certain if the dev issue gets used outside of our team, but I think with the addition of feature request, bugs + features would cover ground there.

**Comment by elycheea** [comment]
For anyone else reviewing — you can preview the form by viewing the file under the Files tab.

image

File: .github/ISSUE_TEMPLATE/feature-request.yaml (hunk 1 above)
Comment (annawen1) [reply .github/ISSUE_TEMPLATE/feature-request.yaml]: yeah good point, I'll remove the requirement in the case for dev feature requests

File: .github/ISSUE_TEMPLATE/feature-request.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      description: 'Provide name for owner/main maintainer(s)'
+    validations:
+      required: true
+  - type: input
+    id: backup-maintainer
+    attributes:
+      label: Second/backup maintainer(s)
+      description: 'Provide name for second/backup maintainer(s)'
+    validations:
+      required: true
```
Comment (annawen1) [inline .github/ISSUE_TEMPLATE/feature-request.yaml:61]: Was thinking on this...is the owner/main maintainer enough that the second/backup maintainer input is optional?

**Review by elycheea — APPROVED** [review]

**Pull request merged**